### PR TITLE
Add support for string arrays

### PR DESCRIPTION
**Pull request by benbardin**
This isn't quite working, what did I miss?

### CODE REVIEW

File: meta/types.go [hunk 1]
```diff
@@ -30,7 +30,8 @@ var Types = []TypeInfo{
 	TypeInfo{Prefix: "NullString", Literal: "sql.NullString"},
 	TypeInfo{Prefix: "NullTime", Literal: "NullableTime"}, // TODO(shutej): test
 	TypeInfo{Prefix: "String", Literal: "string"},
-	TypeInfo{Prefix: "Time", Literal: "time.Time"}, // TODO(shutej): test
+	TypeInfo{Prefix: "StringArray", Literal: "[]string"}, // TODO(benbardin): test
```
Comment (shutej) [inline meta/types.go:33]: is this known to work from your side?  when you can confirm it works, i'll happily merge

Comment (benbardin) [inline meta/types.go:33]: it definitely doesn't. please help :)

Comment (benbardin) [inline meta/types.go:33]: it doesn't seem to know that a varchar[] in a DB is something it can handle, is there another hook i need to add this to?

Comment (shutej) [inline meta/types.go:33]:
```
$ ack varchar
sqlc/generator.go:23:var varchar = regexp.MustCompile("VARCHAR|CHARACTER VARYING|TINYTEXT|TEXT|MEDIUMTEXT|LONGTEXT|CHAR")
sqlc/generator.go:191:                  } else if varchar.MatchString(colType.String) {
sqlc/generator.go:262:                  } else if varchar.MatchString(colType.String) {
```

Comment (shutej) [inline meta/types.go:33]: i don't know that lib/pq will know how to handle this, either, though.  this is a pretty esoteric use of postgres features

Comment (shutej) [inline meta/types.go:33]: in other words, this enterprise is fraught with difficulty.  the 1:m approach can be supported by everything with ease, though.

**Pull request closed without merging**